plugins/newsdownloader.koplugin/feed_view.lua

```diff
@@ -63,10 +63,10 @@ function FeedView:getItem(id, feed, edit_feed_callback, delete_feed_callback)
     end
 
     -- Collect this stuff for later, with the single view.
-    local download_full_article = feed.download_full_article == nil
-        or feed.download_full_article
-    local include_images = not never_download_images
-        and feed.include_images
+    local download_full_article = feed.download_full_article
+        or feed.download_full_article == nil
```

Comment:
This could be more cleanly written as:
`local download_full_article = feed.download_full_article ~= false`
if I understand right that it being undefined (nil) is as if it was true.